Sources/BraintreePaymentFlow/Public/BraintreePaymentFlow/BTLocalPaymentRequest.h

```diff
@@ -76,6 +76,11 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, getter=isShippingAddressRequired) BOOL shippingAddressRequired;
 
+/**
+ Optional: Bank Identification Code of the customer (specific to iDEAL transactions).
+ */
+@property (nonatomic, nullable, copy) NSString *bic;
```

Comment:
Do we have any information on the expected format for this? Numeric, length, alphanumeric etc?

Reply:
The real-life example we've received is alphanumeric

Reply:
> The BIC is an 8 character code, defined as 'business party identifier', consisting of the business party prefix (4 alphanumeric), the country code as defined in ISO 3166-1 (2 alphabetic), and the business party suffix (2 alphanumeric).

Reply:
@scannillo Were you thinking we should call this out in the doc string? Personally, I think it's fine without it.